### PR TITLE
fix mispell and refactor

### PR DESCRIPTION
**Pull request by katherinelc321**
This PR fixes some misspellings and addresses some warnings from the linters https://issues.redhat.com/browse/OSD-9033

### CODE REVIEW

File: pkg/awsManager/route53_manager.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func CleanUpAwsRoute53(client clientpkg.Client, logger logr.Logger) error {
 		}
 	}
 
-	if errFlag == true {
+	if !errFlag {
```
Comment (fahlmant) [inline pkg/awsManager/route53_manager.go:91]: I'm a bit confused about this one. Why return an error if there is no error?

**Comment by dastergon** [comment]
@katherinelc321 can you also write a short description of your PR?

File: pkg/awsManager/aws_manager_test.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/openshift/aws-account-shredder/pkg/localMetrics"
 	"github.com/openshift/aws-account-shredder/pkg/mock"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
```
Comment (dastergon) [inline pkg/awsManager/aws_manager_test.go:19]: Did you verify that this change didn't break the current API?

Comment (katherinelc321) [reply pkg/awsManager/aws_manager_test.go:19]: it does pass the unit tests

Comment (katherinelc321) [reply pkg/awsManager/aws_manager_test.go:19]: just ran the shredder and no errors

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**